tmcprototype/SubarrayNode/SubarrayNode.xmi

```diff
@@ -59,6 +59,16 @@
       <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
       <DefaultPropValue>ska_mid/tm_leaf_node/sdp_subarray01</DefaultPropValue>
     </deviceProperties>
+    <deviceProperties name="CspSubarrayNodeFQDN" description="This property stores the FQDN of CSP subarray Node device to which the Subarray Node is associated with.">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+      <DefaultPropValue>mid_csp/elt/subarray_01</DefaultPropValue>
+    </deviceProperties>
+    <deviceProperties name="SdpSubarrayNodeFQDN" description="This property stores the FQDN of SDP subarray Node device to which the Subarray Node is associated with.">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+      <DefaultPropValue>mid_sdp/elt/subarray_1</DefaultPropValue>
```

Comment:
I see that the naming convention for CSPSubarray follows _01 while that of SDPSubarray follows _1. This is not consistent but can be changed gradually.

Reply:
The CSP subarray device name is provided by CSP team and SDP subarray name is according to the device naming convention.

Reply:
Okay, I think we need to make CSP team aware of the actual naming convention.